modules/nf-core/seqkit/sort/main.nf

```diff
@@ -0,0 +1,69 @@
+process SEQKIT_SORT {
+    tag "$meta.id"
+    label 'process_low'
+    // File IO can be a bottleneck. See: https://bioinf.shenwei.me/seqkit/usage/#parallelization-of-cpu-intensive-jobs
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/seqkit:2.6.1--h9ee0642_0':
+        'biocontainers/seqkit:2.6.1--h9ee0642_0' }"
+
+    input:
+    tuple val(meta), path(fastx)
+
+    output:
+    tuple val(meta), path("${prefix}.*")    , emit: fastx
+    path "versions.yml"                     , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args        = task.ext.args ?: ''
+    def args2       = task.ext.args2 ?: ''
+    prefix          = task.ext.prefix ?: "${meta.id}"
+    def extension   = "fastq"
+    if ("$fastx" ==~ /.+\.fasta|.+\.fasta.gz|.+\.fa|.+\.fa.gz|.+\.fas|.+\.fas.gz|.+\.fna|.+\.fna.gz|.+\.fsa|.+\.fsa.gz/ ) {
+        extension   = "fasta"
+    }
+    extension       = fastx.toString().endsWith('.gz') ? "${extension}.gz" : extension
+    def call_gzip   = extension.endsWith('.gz') ? "| gzip -c $args2 " : ''
+    if("${prefix}.${extension}" == "$fastx") {
+        error "The name of the input file can't be the same as for the output prefix in the " +
+        "module SEQKIT_SORT (currently `$prefix`). Please choose a different one."
+    }
+    """
+    seqkit \\
+        sort \\
+        --threads $task.cpus \\
+        $args \\
+        $fastx \\
+        $call_gzip \\
+        > ${prefix}.${extension}
+
+    cat <<-END_VERSIONS > versions.yml
+    "${task.process}":
+        seqkit: \$(seqkit version | cut -d' ' -f2)
+    END_VERSIONS
+    """
+
+    stub:
+    prefix          = task.ext.prefix ?: "${meta.id}"
+    def extension   = "fastq"
+    if ("$fastx" ==~ /.+\.fasta|.+\.fasta.gz|.+\.fa|.+\.fa.gz|.+\.fas|.+\.fas.gz|.+\.fna|.+\.fna.gz|.+\.fsa|.+\.fsa.gz/ ) {
+        extension   = "fasta"
+    }
+    extension       = fastx.toString().endsWith('.gz') ? "${extension}.gz" : extension
+    if("${prefix}.${extension}" == "$fastx") {
+        error "The name of the input file can't be the same as for the output prefix in the " +
+        "module SEQKIT_SORT (currently `$prefix`). Please choose a different one."
```

Comment:
What is the wording we use for e.g. samtools sort?

Reply:
Thank you! I have changed the wording to make it similar to samtools/sort